src/components/UI/Cart/OrderSummary/DisheItems/DishItem.jsx

```diff
@@ -0,0 +1,75 @@
+import { useContext } from "react";
+import "./DishItem.css";
+import { CartContext } from "../../../../../contexts/Cart/CartContext";
+import EmptyCart from "./Emptycart";
+const DishItem = () => {
+  const {
+    cartState: { cartItems },
+    cartDispatch,
+  } = useContext(CartContext);
+
+  return (
+    <>
+      {cartItems &&
+        cartItems.map((e,i) => {
```

Comment:
While you are applying map method, don't just write "e" every time instead of it use something useful name.